concent_api/sci_testing_common.py

```diff
@@ -0,0 +1,261 @@
+                self.provider_sci.get_gnt_balance(self.provider_sci.get_eth_address()) == 0 or
+                self.requestor_sci.get_gate_address() is None or
+                self.provider_sci.get_gate_address() is None
+            ),
+            timeout_message='Test GNT faucet timeout',
+            sleep_message='Waiting for test GNT for Provider and Requestor...',
+        )
+
+        self._test_gnt_to_gntb_transfer(self.requestor_sci)
+        self._test_gnt_to_gntb_transfer(self.provider_sci)
+
+        self._wait_unitl_timeout(
+            condition=lambda: (
+                self.requestor_sci.get_gntb_balance(self.requestor_sci.get_eth_address()) == 0 or
+                self.provider_sci.get_gntb_balance(self.provider_sci.get_eth_address()) == 0
+            ),
+            timeout_message='Test GNT to GNTB transfer timeout',
+            sleep_message='Waiting for test GNTB for Provider and Requestor...',
+        )
+
+    def request_for_deposit(self, value: int=0):
+        if value < 0:
+            raise ValueNotPositiveError('Value must be greater than zero')
+        requestor_gntb_balance = self.requestor_sci.get_gntb_balance(self.requestor_sci.get_eth_address())
+        provider_gntb_balance = self.provider_sci.get_gntb_balance(self.provider_sci.get_eth_address())
+        if requestor_gntb_balance == 0 or value > requestor_gntb_balance:
+            raise NotEnoughFoundsError("Not enough GNTB founds on requestor's account")
+
+        if provider_gntb_balance == 0 or value > provider_gntb_balance:
+            raise NotEnoughFoundsError("Not enough GNTB founds on provider's account")
+
+        self.requestor_sci.deposit_payment(
+            value if value != 0 else requestor_gntb_balance
+        )
+        self.provider_sci.deposit_payment(
+            value if value != 0 else provider_gntb_balance
+        )
+        self._wait_unitl_timeout(
+            condition=lambda: (
+                self.get_requestor_deposit_value() == 0 or
+                self.get_provider_deposit_value() == 0
+            ),
+            timeout_message='Test deposit GNTB timeout',
+            sleep_message='Waiting for deposited GNTB for Provider and Requestor...',
+        )
+
+    def _get_gntb_balance(self, sci: SmartContractsInterface) -> int:
+        return sci.get_gntb_balance(sci.get_eth_address())
+
+    def get_provider_gntb_balance(self) -> int:
+        return self._get_gntb_balance(self.provider_sci)
+
+    def get_requestor_deposit_value(self):
+        return self.requestor_sci.get_deposit_value(self.requestor_sci.get_eth_address())
+
+    def get_provider_deposit_value(self):
+        return self.provider_sci.get_deposit_value(self.provider_sci.get_eth_address())
+
+    def ensure_that_provider_has_specific_gntb_balance(self, value):
+        self._wait_unitl_timeout(
+            condition=lambda: (
+                    self.get_provider_gntb_balance() != value
```

Comment:
Please fix indent.

Reply:
Done

Reply:
Actually it can fit into single line.

Reply:
Ok, all statement where them could be in single line fixed.